packages/vite/src/node/config.ts

```diff
@@ -838,8 +838,9 @@ export function sortUserPlugins(
 
   if (plugins) {
     plugins.flat().forEach((p) => {
-      if (p.enforce === 'pre') prePlugins.push(p)
-      else if (p.enforce === 'post') postPlugins.push(p)
+      const order = p.order || p.enforce
```

Comment:
In case brillout convinces us to use numbers for order 👀
```suggestion
      const order = p.order ?? p.enforce
```